device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceTwin/DeviceTwin.java

```diff
@@ -69,7 +69,7 @@ public IotHubMessageResult execute(Message message, Object callbackContext)
                 IotHubStatusCode iotHubStatus = IotHubStatusCode.ERROR;
                 if (message.getMessageType() != MessageType.DEVICE_TWIN)
                 {
-                    System.out.print("Unexpected message type received");
```

Comment:
Totally unrelated to this PR, but thought I'd fix this since I saw it. Never use System.out.println from our library